internal/controllers/import_controller.go

```diff
@@ -303,27 +306,21 @@ func (r *CAPIImportReconciler) shouldAutoImport(ctx context.Context, capiCluster
 	return autoImport, nil
 }
 
-func (r *CAPIImportReconciler) reconcileDelete(ctx context.Context, capiCluster *clusterv1.Cluster,
-	rancherCluster *rancher.Cluster,
-) (ctrl.Result, error) {
+func (r *CAPIImportReconciler) reconcileDelete(ctx context.Context, capiCluster *clusterv1.Cluster) (ctrl.Result, error) {
 	log := log.FromContext(ctx)
-	log.Info("Reconciling CAPI Cluster Delete")
+	log.Info("Reconciling rancher cluster deletion")
```

Comment:
but we are reconciling CAPI cluster deletion

Reply:
Right now we have to act upon the fact that rancher cluster is getting deleted